src/bindist/config.sys

```diff
@@ -9,6 +7,6 @@ buffers=10
 devicehigh=e:\dosemu\ems.sys
 devicehigh=e:\dosemu\cdrom.sys
 rem uncomment to load another bitmap font (see also autoemu.bat)
-rem devicehigh=c:\dos\display.sys con=(ega,,1)
+rem devicehigh=d:\dos\display.sys con=(ega,,1)

```

Comment:
That doesn't produce an error, but the nlsfunc error for the chcp command stays. I don't really know what exactly each command sets up around the codepage (and why multiple are needed).